lib/grape/app/templates/Rakefile

```diff
@@ -1,6 +1,22 @@
 require 'bundler/setup'
-require 'rspec/core/rake_task'
 require 'grape/app/tasks'
 
-RSpec::Core::RakeTask.new(:spec)
+begin
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+  Rake::Task['spec'].enhance ['db:test:prepare']
```

Comment:
please remove that line. we do have "maintain schema" in spec helper, it should work. if not, we should not workaround like this

Reply:
Hint for rspec spec_helper:

```
  config.before :suite do
    ActiveRecord::Migration.maintain_test_schema!
  end
```

Reply:
`if defined?(ActiveRecord)`